routers/api/v2/tokens.go

```diff
@@ -0,0 +1,64 @@
+package v2
+
+import (
+	"github.com/gin-gonic/gin"
+	v2 "github.com/unicsmcr/hs_auth/authorization/v2"
+	"github.com/unicsmcr/hs_auth/routers/api/models"
+	"go.uber.org/zap"
+	"net/http"
+	"strconv"
+	"strings"
+)
+
+func (r *apiV2Router) ServiceToken(ctx *gin.Context) {
```

Comment:
```suggestion
func (r *apiV2Router) CreateServiceToken(ctx *gin.Context) {
```

Also, could you add an operation doc like this? https://github.com/unicsmcr/hs_auth/blob/master/routers/api/v2/users.go#L12-L17

Reply:
Ah good point 🙂